docs/layouts/shortcodes/generated/spark_connector_configuration.html

```diff
@@ -56,6 +56,12 @@
             <td>Long</td>
             <td>The minimum number of rows returned in a single batch, which used to create MinRowsReadLimit with read.stream.maxTriggerDelayMs together.</td>
         </tr>
+        <tr>
+            <td><h5>read.changeLog</h5></td>
```

Comment:
In general, the key of option does not need to be capitalized, let's use lowercase here